<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>524</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Otimizar padrão de fase</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.xpm</normaloff>icon.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QGroupBox" name="gbCamera">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Câmera</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="lblCamera">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="gbCameraSideView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>Vista lateral</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="2">
        <widget class="QLabel" name="lblIn">
         <property name="text">
          <string>em</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbPosition">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>X</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Y</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="QLabel" name="lblCameraSideView">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblPosition">
         <property name="text">
          <string>Posição</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="sbPositionValue">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1280</number>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="gbPhase">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Padrão de fase</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QLabel" name="lblX">
         <property name="text">
          <string>Posição X:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="lblConfidence">
         <property name="text">
          <string>Nível de confidência: </string>
         </property>
        </widget>
       </item>
       <item row="7" column="6">
        <widget class="QPushButton" name="pbOptimize">
         <property name="text">
          <string>&amp;Otimizar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="6">
        <widget class="QLabel" name="lblPhase">
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="sbX">
         <property name="maximumSize">
          <size>
           <width>52</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Indica a posição X do padrão de fase no canvas do SLM&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1280</number>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QSpinBox" name="sbY">
         <property name="maximumSize">
          <size>
           <width>52</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Indica a posição Y do padrão de fase no canvas do SLM&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="pbAlign">
         <property name="text">
          <string>A&amp;linhar</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="6">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="lblY">
         <property name="text">
          <string>Posição Y:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="5">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="sbDistanceGrating">
         <property name="maximumSize">
          <size>
           <width>52</width>
           <height>32</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QSpinBox" name="sbLengthGrating">
         <property name="maximumSize">
          <size>
           <width>52</width>
           <height>32</height>
          </size>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>1024</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lblDistanceGrating">
         <property name="text">
          <string>Distância:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="lblLengthGrating">
         <property name="text">
          <string>Comprimento:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QCheckBox" name="cbBinary">
         <property name="text">
          <string>Imagem binária</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="5" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="mnFile">
    <property name="title">
     <string>&amp;Arquivo</string>
    </property>
    <addaction name="actLoadImage"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="mnHelp">
    <property name="title">
     <string>A&amp;juda</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="mnFile"/>
   <addaction name="mnHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actQuit">
   <property name="text">
    <string>&amp;Sair</string>
   </property>
  </action>
  <action name="actLoadImage">
   <property name="text">
    <string>&amp;Carregar imagem</string>
   </property>
  </action>
  <action name="actAbout">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actSaveImage">
   <property name="text">
    <string>Salvar imagem</string>
   </property>
   <property name="toolTip">
    <string>Salvar imagem</string>
   </property>
  </action>
  <action name="actSavePhase">
   <property name="text">
    <string>Salvar fase</string>
   </property>
  </action>
  <action name="actSaveGraphic">
   <property name="text">
    <string>Salvar gráfico</string>
   </property>
  </action>
  <action name="actUpdatePhase">
   <property name="text">
    <string>Atualizar</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sbX</tabstop>
  <tabstop>sbY</tabstop>
  <tabstop>sbDistanceGrating</tabstop>
  <tabstop>sbLengthGrating</tabstop>
  <tabstop>cbBinary</tabstop>
  <tabstop>pbAlign</tabstop>
  <tabstop>pbOptimize</tabstop>
  <tabstop>cbPosition</tabstop>
  <tabstop>sbPositionValue</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
